configure.ac

```diff
@@ -568,7 +568,7 @@ AS_CASE([$ocaml_cv_cc_vendor],
   cc_warnings='-Wall -Wdeclaration-after-statement'])
 
 AS_CASE([$enable_warn_error,AC_PACKAGE_VERSION],
-  [yes,*|,*+dev*|,*+multicore*],
+  [yes,*|,*+dev*|,*+domains*],
```

Comment:
Wouldn't we need to also update the string in the `VERSION` file to match?

There is a decision here, if we change the `VERSION`:
- we might break things downstream expecting `+multicore` in the string (might not be a concern)
- we would want different version strings for `+domains` and `+domains+effects` on the branches

Reply:
Yes - I think originally that was expected in another PR... it's now done here too!